scripts/actions/mobskills/magma_fan.lua

```diff
@@ -10,7 +10,7 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local dmgmod = xi.mobskills.mobBreathMove(mob, target, skill, 0.5, 1.25, xi.element.FIRE, 600)
+    local dmgmod = xi.mobskills.mobBreathMove(mob, target, skill, 0.0833, 1, xi.element.FIRE, 600)
```

Comment:
0.0833 seems a lil weird. I would just put `1 / 12` Just an opinion though, it can stay that way

Reply:
Yeah, I could go either way. It is one less operation but negligible. Maybe someone else has a stronger opinion on the matter.

Reply:
In my case its more a matter of some1 comming, seeing it and asking "wtf does this number even mean"
As a fraction, on the other hand, seems like a more reasonable number.

But as I said, not really a deal breaker to me.

Reply:
Perhaps a compromise with a comment?  I'm sure its documented somewhere, and could help alleviate future concerns (should one arise)

Reply:
added a comment :)